common/tls/src/main/java/io/helidon/common/tls/TlsConfigBlueprint.java

```diff
@@ -250,31 +254,33 @@ static List<X509Certificate> createTrust(Keys config) {
      *
      * @return session cache size, defaults to 1024
      */
-    @ConfiguredOption("1024")
+    @Option.Configured
+    @Option.DefaultInt(1024)
     int sessionCacheSize();
 
     /**
      * SSL session timeout.
      *
      * @return session timeout, defaults to 30 minutes
      */
-    @ConfiguredOption("PT30M")
+    @Option.Configured
+    @Option.Default("PT30M")
     Duration sessionTimeout();
```

Comment:
Did we change the default value from `0` between Helidon 3 and 4?

Reply:
Seems like it. The default in Java is 24 hours

Reply:
Created #8127